### PR TITLE
add Elasticsearch datasource

### PR DESCRIPTION
**Pull request by duzvik**
adding Elasticsearch datasource

### CODE REVIEW

File: beagle/datasources/elasticsearch_qs.py [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+        query : str
+            Elasticsearch query string, by default "*"
+        earilest : str, optional
+            The earliest time modifier, by default "-7d"
+        latest : str, optional
+            The latest time modifier, by default "now"
+        """
+
+        self.earliest = earliest
+        self.latest = latest
```
Comment (yampelo) [inline beagle/datasources/elasticsearch_qs.py]: Does latest actually get used?

**Comment by yampelo** [comment]
Can you please check this button on the PR so i can push to it?

image

**Comment by duzvik** [comment]
Checkbox is "On"
Знімок екрана  о 09 42 44

File: beagle/datasources/elasticsearch_qs.py (hunk 1 above)
Comment (duzvik) [reply beagle/datasources/elasticsearch_qs.py]: fixed.

**Pull request merged**